Packs/FancyEmails/Scripts/MakeFancyEmailIndicatorTable/MakeFancyEmailIndicatorTable.py

```diff
@@ -0,0 +1,90 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+query = demisto.args().get('query')
+name = demisto.args().get('name')
+investigation_url = demisto.demistoUrls().get('investigation')
+use_alt_link = demisto.args().get('use_alt_link') in ['True', 'true']
+max_name_characters = int(demisto.args().get('max_name_characters', 40))
```

Comment:
We have a function that handles this gracefully for you in CommonServerPython, called arg_to_number.